test/analyzer_test.js

```diff
@@ -116,8 +154,8 @@ suite('Analyzer', () => {
       })
       return analyzer.getEntities(document).then((entities) => {
         assert.equal(entities.length, 2);
-        assert.instanceOf(entities[0], DocumentDescriptor);
-        assert.instanceOf(entities[1], DocumentDescriptor);
```

Comment:
Consider using `chai-subset` to quickly assert a bunch of things in an easy to use way and future-unobtrusive way:

```javascript
expect(entities[0]).to.containSubset({type: 'js', contents: `console.log('hi')`});
expect(entities[1]).to.containSubset({type: 'css', contents: `body { color: red; }`});
```